mender/values.yaml

```diff
@@ -62,11 +62,12 @@ mongodb:
   image:
     tag: "4.4.13-debian-10-r63"
   auth:
-    enabled: false
-    username: ""
-    password: ""
+    enabled: true
+    rootPassword: "pleasechangeme"
+    replicaSetKey: "pleasechangeme"
+    username: "mender"
     # reference to Bitnami chart only: the database name is managed by Mender only
-    database: ""
+    database: "mender"
```

Comment:
This username/database does not make sense as I believe these are restricted to a single `database`. In our case, where we share credentials across all microservices, we only need in the `rootPassword` and `replicaSetKey`.

Reply:
That was my first iteration, but then the bitnami chart complained for missing `database` object :shrug: .
That's why I set the comment: `# reference to Bitnami chart only: the database name is managed by Mender only`